book/B-embedding-git/sections/dulwich.asc

```diff
@@ -0,0 +1,44 @@
+=== Dulwich
+
+(((Dulwich)))((("Python")))
```

Comment:
Interesting. Does this index entry require the quotes?

Reply:
Do not know, to be hones, I just copied it from libgit2 example. 
Digging deeper, https://github.com/progit/progit2/pull/471 seems to be a reason for those quotes.

Will be happy to remove those if you think there might be some side-effects, but I guess same logic should be applicable for [go-git](https://github.com/progit/progit2/blame/master/book/B-embedding-git/sections/go-git.asc#L3) as well.

Let me know if you want me to remove both in this PR

Reply:
Alright, let's maybe clean them up in a separate PR then.

Reply:
👍 https://github.com/progit/progit2/pull/1096 created